sdk/src/test/java/io/dapr/client/DaprPreviewClientGrpcTest.java

```diff
@@ -194,9 +198,11 @@ public void subscribeConfigurationTest() {
 			return null;
 		}).when(daprStub).subscribeConfigurationAlpha1(any(DaprProtos.SubscribeConfigurationRequest.class), any());
 
-		Iterator<List<ConfigurationItem>> itr = previewClient.subscribeToConfiguration(CONFIG_STORE_NAME, "configkey1").toIterable().iterator();
+		Iterator<SubscribeConfigurationResponse> itr = previewClient.subscribeToConfiguration(CONFIG_STORE_NAME, "configkey1").toIterable().iterator();
 		assertTrue(itr.hasNext());
-		assertEquals("configkey1", itr.next().get(0).getKey());
+		SubscribeConfigurationResponse res = itr.next();

```

Comment:
best to check it has next and then calling next.

Reply:
Actually the test is based on this logic that the next item has to exist. See line just above this `assertTrue(itr.hasNext());`. If the next item is not there the test should fail.